cdap-examples/HelloWorld/src/test/java/co/cask/cdap/examples/helloworld/HelloWorldTest.java

```diff
@@ -44,6 +45,10 @@
 
   @Test
   public void test() throws Exception {
+    NamespaceId streamSpace = new NamespaceId("streamSpace");
```

Comment:
We should deploy to a different namespace right? That's the way to test cross namespace access.